pages/documentation/showcase.md

```diff
@@ -48,138 +52,136 @@ If your project is currently using USWDS and you would like to see it included i
 - [General Services Administration](https://gsa.gov)
-- [General Services Administration - Tech at GSA](https://tech.gsa.gov/)
 - [GI Bill Comparison Tool (Vets.gov)](https://www.vets.gov/gi-bill-comparison-tool)
+- [Glenn Research Center (NASA)](https://www1.grc.nasa.gov)
 - [GSA Grace Hopper Day Hackathon](https://open.gsa.gov/events/grace-hopper-hackathon/)
 - [openGSA - GSA Open Technology](https://open.gsa.gov/)
 - [HealthCare.gov](https://www.healthcare.gov/)
 - [Hack the Pay Gap](https://paygap.pif.gov/)
 - [Healthcare.gov Tax Tool](https://www.healthcare.gov/tax-tool/)
+- [HelpWithMyBank.gov](https://helpwithmybank.gov/)
 - [Home Mortgage Disclosure Act](https://ffiec.cfpb.gov/)
 - [The HTTPS-Only Standard](https://https.cio.gov/)
+- [Human Computer Interaction Group (NASA)](https://hci.arc.nasa.gov)
 - [Institute of Museum and Library Services](https://imls.gov/)
 - [Interagency Working Group on U.S. Government-Sponsored International Exchanges and Training (IAWG)](https://iawg.gov/)
 - [IT Dashboard](https://itdashboard.gov/)
-- [Join an NIEHS Study - National Institute of Environmental Health Sciences](https://joinastudy.niehs.nih.gov/)
-- [LINCS - Adult Education and Literacy - U.S. Department of Education](https://lincs.ed.gov/)
+- [Join an NIEHS Study (National Institute of Environmental Health Sciences - NIH)](https://joinastudy.niehs.nih.gov/)
+- [LINCS (Adult Education and Literacy - ED)](https://lincs.ed.gov/)
 - [Legislative Branch Innovation Hub](https://usgpo.github.io/innovation/)
 - [Login.gov](https://www.login.gov/)
-- [Manufacturing USA](https://www.manufacturing.gov/)
+- [Manufacturing.gov](https://www.manufacturing.gov/)
 - [Medicaid](https://www.medicaid.gov/)
-- [Method Cards - 18F](https://methods.18f.gov/)
+- [Method Cards (18F)](https://methods.18f.gov/)
 - [Move.mil — Official DOD Moving Portal](https://www.move.mil/)
 - [MyMedicare](https://www.mymedicare.gov/)
+- [National Agricultural Statistics Service (USDA)](https://release.nass.usda.gov/)
 - [NASA General Coordinates Network (GCN)](https://gcn.nasa.gov)
-- [NASA Glenn Research Center](https://www1.grc.nasa.gov)
-- [NASA Human Computer Interaction Group](https://hci.arc.nasa.gov)
-- [National Agricultural Statistics Service - USDA](https://release.nass.usda.gov/)
 - [National Archives and Records Administration](https://archives.gov)
 - [National Archives Museum](https://museum.archives.gov/)
 - [National Blue Ribbon Schools](https://nationalblueribbonschools.ed.gov)
 - [National Environmental Sattelite Data and Information Service (NOAA)](https://www.nesdis.noaa.gov/)
-- [National Initiative for Cybersecurity Careers & Studies (NICCS)](https://niccs.us-cert.gov/)
+- [National Initiative for Cybersecurity Careers & Studies](https://niccs.us-cert.gov/)
 - [National Institutes of Health](https://www.nih.gov/)
 - [The National Flood Insurance Program (FloodSmart)](https://www.floodsmart.gov/)
```

Comment:
Should this be moved to maintain alphabetical order?

Reply:
@finekatie @sarah-sch I updated the order of this in [9ec848c](https://github.com/uswds/uswds-site/pull/2272/commits/9ec848c8587323c5b16f5a8c45bed06395f303bc). Let me know if you need any changes!

Reply:
Hi @amyleadem , I'd leave the acronym where it was. Not sure why it was removed. :)

Reply:
@amyleadem, updates for alphabetical order in [9ec848c](https://github.com/uswds/uswds-site/pull/2272/commits/9ec848c8587323c5b16f5a8c45bed06395f303bc) look good--thanks!